src/pages/HomeWarming/Comment.jsx

```diff
@@ -0,0 +1,14 @@
+import * as S from './Comment.Style';
+
+export default function Comment() {
+  return (
+    <S.CommentWrap>
+      <S.CountComment>댓글갯수</S.CountComment>
+      <S.InputWrap>
+        <S.UserImg src="./images/HomeWarming/userImg.png" alt="userimg" />
```

Comment:
images는 절대경로로 접근해야 됩니다~
지금의 방법은 상대경로로 접근하셨어요
```suggestion
        <S.UserImg src="/images/HomeWarming/userImg.png" alt="userimg" />
```

Reply:
넵 반영하겠습니다!